apps/agent/app/src/components/AgentHelp.js

```diff
@@ -25,19 +21,24 @@ function InstallFrame() {
             align-items: center;
           `}
         >
-          <div>
-            <FrameSvg />
+          <div
+            css={`
+              width: 46px;
+              height: 46px;
```

Comment:
As a rule we avoid setting sizes with numbers, and always use the `GU` import with corresponds to 8 pixels, like this: `width: ${6 * GU}px`. This way we're more consistent. Let's GUify this!

Reply:
Ah, I see. Done!